src/battle_script_commands.c

```diff
@@ -11224,6 +11224,54 @@ static void Cmd_various(void)
         gBattlescriptCurrInstr = cmd->nextInstr;
         return;
     }
+    case VARIOUS_TRY_REVIVAL_BLESSING:
+    {
+        VARIOUS_ARGS(const u8 *jumpInstr);
+        u32 side = GetBattlerSide(gBattlerAttacker);
+        u8 index = GetFirstFaintedPartyIndex(gBattlerAttacker);
+
+        // Move fails if there are no battlers to revive.
+        if (index == PARTY_SIZE)
+        {
+            gBattlescriptCurrInstr = cmd->jumpInstr;
+            return;
+        }
+
+        // Trainer AI chooses first fainted battler, no need to call controllers.
```

Comment:
I wonder if not calling the controllers means that the move won't work in a link battle?

Reply:
This is a great question and one I don't really know the answer, too. I have never had to think about link battle compatibility, honestly. I do know there is a missing call to create a standby message for link battles when the player opens the party that U-turn has, but when I put it in things don't work :/ I can try to work at it if we want it to work!